extensions/account-registration.md

```diff
@@ -56,13 +56,13 @@ a compatible work-in-progress version.
 
 ### Commands
 
-    REGISTER <accountname> {<email> | "*"} <password>
+    REGISTER <account> {<email> | "*"} <password>
```

Comment:
this should probably be left as accountname so that it's more clear

Reply:
The rest of the spec uses `<account>`.